websites/#/アイドルマスターシャイニーカラーズ/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "258380809172418570",
+		"name": "sino"
+	},
+	"service": "アイドルマスターシャイニーカラーズ",
+	"description": {
```

Comment:
This service could do with an `altnames` field (romanization, english translation, shorter names, etc.)